src/shim.c

```diff
@@ -1880,7 +1895,7 @@ EFI_STATUS start_image(EFI_HANDLE image_handle, CHAR16 *ImagePath)
 	 */
 	efi_status = handle_image(data, datasize, li, &entry_point,
 				  &alloc_address, &alloc_pages);
-	if (EFI_ERROR(efi_status)) {
+	if (efi_status != EFI_SUCCESS) {
```

Comment:
This change looks not necessary.

Reply:
It's explained in the commit message - without this change, build fails with:

.././src/shim.c: In function ‘start_image’:
.././src/shim.c:1966:15: error: ‘entry_point’ may be used uninitialized in this function [-Werror=maybe-uninitialized]

Reply:
Ah, thanks. Didn't realize this it related to the error mentioned in the commit message.

Reply:
I'll rewrite the commit message to clarify that, thank you!